site/src/render.tsx

```diff
@@ -27,7 +27,9 @@ const skuRender: Render<RenderContext> = {
     const versionMap = await braidVersionToDate();
 
     const sourceUrlPrefix = `${githubUrl}${prSha || 'master'}`;
-    const routerBasename = isGithubPages ? 'braid-design-system' : '';
+    const routerBasename = isGithubPages
+      ? 'braid-design-system/lib/components'
```

Comment:
This looks like it might have got caught in a find replace. I'd think this should be rolled back.

Reply:
🤦